src/components/Extensions/Details/DetailCard/Tabs/ParamList.tsx

```diff
@@ -81,6 +173,22 @@ function ParamList() {
           </div>
         );
       })}
+      <div className={styles.paramWrapper}>
+        <h4>Enabled Events</h4>
+        <Typography
+          className={styles.description}
+          use="caption"
+          tag="div"
+          theme="secondary"
+        >
+          If you enable events, you can <a href="https://firebase.google.com/docs/extensions/install-extensions#eventarc">write custom event handlers</a> that respond to these events. 
+          Events will be emitted via Eventarc. <a href="https://cloud.google.com/eventarc/pricing)">Fees apply.</a>
+          <br />
+          <a href={`localhost:3000/extensions/${extension.id}`}>How do events in this extension work?</a>
```

Comment:
todo: Customize the port?
todo: Can't use query parameters (like ?tab=usage) to navigate around pages. May need to use the router to do this?

Reply:
Actually, I realized in this case it doesn't matter, we just need to redirect them to the starting state. So clicking the link works fine.